figure_notebooks/supp_country_specific_analysis/country_specific_analysis.md

```diff
@@ -0,0 +1,891 @@
+       quote_springer_indiv_sub_gg, width = 7, height = 5, units = "in", device = "pdf")
+
+mention_springer_indiv_sub_gg = ggplot(subset(all_bootstrap_df, 
+                                         corpus %in% c("springer_last", 
+                                                       "mention") &
+                                         name_origin %in% c("CelticEnglish", "EastAsian", "European")), 
+                                aes(x=as.numeric(year), y=mean,
+                                      ymin=bottom_CI, ymax=top_CI,
+                                      fill=corpus)) +
+    geom_point() + geom_ribbon(alpha=0.5) + geom_line(alpha=0.5) +
+    theme_bw() + 
+    xlab("Year of Article") + ylab("Percentage mentions or Articles") +
+    ggtitle(paste("Prop. mention vs Last Authorship by Name Origin -- restricted to US affil.")) + 
+    scale_fill_manual(values=QUOTE_ANALYSIS_COLOR) +
+    facet_wrap(~ name_origin, dir="h", scales="free") +
+    theme(legend.position="bottom")
+
+ggsave(file.path(proj_dir, "/figure_notebooks/supp_country_specific_analysis/tmp/mention_springer_indiv_sub_gg.pdf"),
+       mention_springer_indiv_sub_gg, width = 7, height = 5, units = "in", device = "pdf")
+```
+
+### format supp. figure
+
+``` r
+quote_cited_nature_indiv_sub_gg = image_read_pdf(file.path(proj_dir,
+                                  "/figure_notebooks/supp_country_specific_analysis/tmp/quote_cited_nature_indiv_sub_gg.pdf"))
+quote_cited_nature_indiv_sub_gg = image_annotate(quote_cited_nature_indiv_sub_gg, "a", size = 30)
+
+
+quote_name_origin_us_gg = image_read_pdf(file.path(proj_dir,
+                                  "/figure_notebooks/supp_country_specific_analysis/tmp/quote_nature_indiv_sub_gg.pdf"))
+quote_name_origin_us_gg = image_annotate(quote_name_origin_us_gg, "b", size = 30)
+
+
+mention_cited_nature_indiv_sub_gg = image_read_pdf(file.path(proj_dir,
+                                  "/figure_notebooks/supp_country_specific_analysis/tmp/mention_cited_nature_indiv_sub_gg.pdf"))
+mention_cited_nature_indiv_sub_gg = image_annotate(mention_cited_nature_indiv_sub_gg, "c", size = 30)
+
+
+mention_nature_indiv_sub_gg = image_read_pdf(file.path(proj_dir,
+                                  "/figure_notebooks/supp_country_specific_analysis/tmp/mention_nature_indiv_sub_gg.pdf"))
+mention_nature_indiv_sub_gg = image_annotate(mention_nature_indiv_sub_gg, "d", size = 30)
+
+
+
+top_image <- image_append(image_scale(c(quote_cited_nature_indiv_sub_gg,
+                                           quote_name_origin_us_gg),3000), stack = FALSE)
+
+bottom_image <- image_append(image_scale(c(mention_cited_nature_indiv_sub_gg, 
+                                           mention_nature_indiv_sub_gg),3000), stack = FALSE)
+full_image <- image_append(image_scale(c(top_image, bottom_image), 3000), stack = TRUE)
+
+print(full_image)
+```
+
+    ## # A tibble: 1 x 7
```

Comment:
Interesting to find that the EastAsian group has a greater proportion of last authors than quotes or mentions in 2020.

Reply:
hmmm... I think I'm missing something, could you explain a bit more? It may also be because the y-axis is different for each subpanel, its a bit tricky to see since the numbers are small.